File: Drivers/FSCC Drivers/windows/lib/fscc/net/examples/append-timestamp.cs
using Fscc;
using System;

public class Tutorial
{
    public static int Main(string[] args)
    {
        Fscc.Port p = new Fscc.Port(0);

        var status = p.AppendTimestamp;
        
        p.AppendTimestamp = true;
        p.AppendTimestamp = false;

        return 0;
    }
}

File: Drivers/FSCC Drivers/linux/examples/append-timestamp.c
#include <fcntl.h> /* open, O_RDWR */
#include <unistd.h> /* close */
#include <fscc.h> /* FSCC_* */

int main(void)
{
    int fd = 0;
    unsigned status = 0;

    fd = open("/dev/fscc0", O_RDWR);

    ioctl(fd, FSCC_GET_APPEND_TIMESTAMP, &status);

    ioctl(fd, FSCC_ENABLE_APPEND_TIMESTAMP);
    ioctl(fd, FSCC_DISABLE_APPEND_TIMESTAMP);

    close(fd);

    return 0;
}
